metrics/metrics.go

```diff
@@ -781,3 +852,8 @@ func Timer(ctx context.Context, m *stats.Float64Measure) func() time.Duration {
 		return time.Since(start)
 	}
 }
+
+func AddNetworkTag(ctx context.Context) context.Context {
+	ctx, _ = tag.New(ctx, tag.Upsert(Network, buildconstants.NetworkBundle))
```

Comment:
Thankfully, `ctx` will always be something even in the case of an error, so ignoring the error won't hurt - but it's worth checking these kinds of things out because the risk here is that `ctx` ends up being `nil`: https://github.com/census-instrumentation/opencensus-go/blob/v0.24.0/tag/map.go#L187

Another alternative here worth considering if a `panic` on an error, which might not be a bad thing to do because if we're erroring then something really bad has gone wrong. I think for now we can go with ignoring the error because it's what the rest of the Lotus codebase does so we'll stick to convention for this.